src/features/groups/components/GroupCardList/GroupCard/GroupCardContent/GroupCardDropdownMenu/GroupCardDropdownMenuContent.tsx

```diff
@@ -0,0 +1,46 @@
+import { IconDelete, IconPen } from '@/assets/images/icons';
+import {
+  DropdownMenuButton,
+  DropdownMenuButtonIcon,
+  DropdownMenuButtonText,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  Icon,
+} from '@/components/ui';
+
+interface IGroupCardDropdownMenuContentProps {
+  /**
+   * Function to handle the rename button click.
+   */
+  handleRenameClick: () => void;
+  /**
+   * The function to close the dropdown menu.
+   */
+  handleDeleteClick: () => void;
+}
+
+export const GroupCardDropdownMenuContent = ({
+  handleRenameClick,
+  handleDeleteClick,
+}: IGroupCardDropdownMenuContentProps) => {
+  return (
+    <DropdownMenuContent>
+      <DropdownMenuItem>
```

Comment:
`asChild` should be added so we can remove the unnecessary div and manipulate it only with the keyboard.

Do you mind fixing the ones on the profile page as well?

Reply:
@nick-y-ito Thank you for your review! as we dicussed in person, I leave fixing ones on the profile page and make an another issue.